packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,56 @@
+/* globals JitsiMeetExternalAPI */
+
+Template.videoFlexTab.helpers({
+
+});
+
+Template.videoFlexTab.onCreated(function() {
+	let rid = this.data && this.data.rid;
+	this.timeout = null;
+	this.autorun(() => {
+		if (RocketChat.settings.get('Jitsi_Enabled')) {
+			if (RocketChat.TabBar.getTemplate() === 'videoFlexTab') {
+				if (RocketChat.TabBar.isFlexOpen()) {
+					RocketChat.TabBar.updateButton('video', { class: 'red' });
+				} else {
+					RocketChat.TabBar.updateButton('video', { class: '' });
+
+					if (this.timeout) this.timeout();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'this'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2716979626/issues/source?bid=3199634&fileBranchId=3342079#l18)